meta/main.yml

```
@@ -6,16 +6,12 @@ galaxy_info:
 
   license: CC-BY-SA
 
-  min_ansible_version: 2.0
+  min_ansible_version: 2.7
```

what feature of Ansible 2.7 do you use?
loops and calling apt with a list are supported since 2.5
```suggestion
  min_ansible_version: 2.5
```